### ISSUE TITLE
Outlook: randum occurrences of editing not supported when composing, forwarding, replying

### ISSUE THREAD

**Issue by PratikP1** [nvaccess/nvda#6990]
Test Environment: Outlook 2016, Windows 10 Insider build 15061, NVDA version next-13933,231f5b92

This problem occurs randomly. Seems to have no pattern that I could detect. So I cant give steps to reproduce. I can only provide a description and a log. This has been happening for a while. I haven't reported it  because  I couldn't perceive a pattern.

Occasionally, NVDA starts announcing "Not supported in this document" when trying to type. This happens in any message field, replying, composing a new message, or forwarding. The field is acting as if browse mode navigation is possible. I suspect that NVDA is thinking that the message is being read and not being composed.

Restarting NVDA solves this problem. The errors and warnings in the log might point to a cause.
[NVDA, Outlook, Not Supported In This Document.txt](https://github.com/nvaccess/nvda/files/853302/NVDA.Outlook.Not.Supported.In.This.Document.txt)

**Comment by Adriani90** [comment]
@PratikP1 is this still the case when composing / forwarding a message? I cannot reproduce it in NVDA alpha-17029,4d2e7f2b.

**Comment by Adriani90** [comment]
@PratikP1 could you please test with last NVDA alpha if this is still an issue? As metioned, I still cannot reproduce it. Since we don't have updates on this since years, I am thinking to close this as abandoned and works for me.

**Comment by PratikP1** [comment]
This is no longer a problem. The issue can be closed.
